packages/nimble-components/src/table/specs/table-columns-hld.md

```diff
@@ -147,7 +145,37 @@ abstract class TableColumn<TCellData extends TableRecord = TableRecord, TColumnC
 
 _Note: The `TableColumn` class may be updated to support other features not covered in this HLD such as sorting and grouping._
 
-Given the above class, a series of column elements to handle basic use cases can be written within Nimble. For example, the `TableColumn` implementation we could create for rendering data as a read-only `NimbleTextField` could look like this:
+### `BaseCellElement<>`
```

Comment:
I think using the name element is a bit redundant, I like the term cell view as used by `cellViewTag`. Also our TableColumn base class isn't using `Base` in the name so to be consistent at least for column authors I'd leave it off.

So I'd suggest the name `TableCellView` (it is close to TableCell but I think it's actually not a bad distinction of TableCell vs TableCellView)

Reply:
I'm definitely up for alternate naming proposals. I think the main change I wanted to incorporate from your initial pseudocode was having `Cell` in the name to make it clear that the element will used repeatedly in individual cells - I didn't think something like `[...]ColumnView[...]` conveyed that.

Reply:
Updated to `TableCellView`, and updated the examples in the table columns HLD from `*CellElement` to `*CellView`.